packages/ethereum-contracts/test/contracts/agreements/ConstantFlowAgreementV1.test.ts

```diff
@@ -1,52 +1,65 @@
-const TestEnvironment = require("../../TestEnvironment");
-const {expectCustomError} = require("../../utils/expectRevert");
-const {expectEvent} = require("@openzeppelin/test-helpers");
-const {ethers} = require("hardhat");
-const {expect} = require("chai");
-const {toBN, toWad, max} = require("../utils/helpers");
-const {
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
```

Comment:
while we are at it, let's split this file into multiple ones: testing without callbacks, testing with MFA, and testing with callbacks.